Not being able to find expressworks after restarting c9 workspace

Steps to reproduce:

1. Create a new workspace on c9.
2. Execute the command given by the freeCodeCamp challenge:
  `git clone https://github.com/FreeCodeCamp/fcc-expressworks.git && chmod 744 fcc-expressworks/setup.sh && fcc-expressworks/setup.sh && source ~/.profile`.
3. Try the `expressworks` command to confirm it's working.
4. Restart the workspace(`Cloud9->Restart Workspace`); Or close the tab and wait a few hours(?, minutes maybe) so the workspace is shut down and need to be restarted when you reopen.
5. Try the `expressworks` command again: `bash: expressworks: command not found`.

Bothers me because if you need to leave, in the middle of completing the tests, when you come back the command doesn't work anymore.

My only workaround is creating a new workspace from scratch, leaving behind the exercises I've done. Because trying to reinstall it brings a bunch of problems such as `fatal: destination path 'fcc-expressworks' already exists and is not an empty directory.`; if you remove the `fcc-expressworks` directory it brings up `fatal: destination path 'expressworks' already exists and is not an empty directory.`; if you also remove the `expressworks` directory a bigger error, that I haven't found out how to solve, shows up:

`npm ERR! rofs EROFS, unlink '/nix/store/9vkhrpycs37cl5i8pywqkdz22773bybf-nodejs-0.10.38/lib/node_modules/expressworks'`
`npm ERR! rofs This is most likely not a problem with npm itself`
`npm ERR! rofs and is related to the file system being read-only.`
`npm ERR! rofs `
`npm ERR! rofs Often virtualized file systems, or other file systems`
`npm ERR! rofs that don't support symlinks, give this error.`
`npm ERR! System Linux 4.2.0-c9`
`npm ERR! command "/nix/store/9vkhrpycs37cl5i8pywqkdz22773bybf-nodejs-0.10.38/bin/node" "/mnt/shared/sbin/npm" "install" "-g" "expressworks/"`
`npm ERR! cwd /home/ubuntu/workspace`
`npm ERR! node -v v0.10.28`
`npm ERR! npm -v 1.4.28`
`npm ERR! path /nix/store/9vkhrpycs37cl5i8pywqkdz22773bybf-nodejs-0.10.38/lib/node_modules/expressworks`
`npm ERR! code EROFS`
`npm ERR! errno 56`
`npm ERR! error rolling back Error: EROFS, unlink '/nix/store/9vkhrpycs37cl5i8pywqkdz22773bybf-nodejs-0.10.38/lib/node_modules/expressworks'`
`npm ERR! error rolling back  { [Error: EROFS, unlink '/nix/store/9vkhrpycs37cl5i8pywqkdz22773bybf-nodejs-0.10.38/lib/node_modules/expressworks']`
`npm ERR! error rolling back   errno: 56,`
`npm ERR! error rolling back   code: 'EROFS',`
`npm ERR! error rolling back   path: '/nix/store/9vkhrpycs37cl5i8pywqkdz22773bybf-nodejs-0.10.38/lib/node_modules/expressworks' }`
`npm ERR! not ok code 0`

--- a/setup.sh
+++ b/setup.sh
@@ -5,6 +5,9 @@ echo "Setting up nvm..."
 source ~/.nvm/nvm.sh
 V11_LEVEL=v4.1
 
+# Install node
+nvm install ${V11_LEVEL}
+
 # Set that level as default
 nvm use ${V11_LEVEL}
 

